pages/handoff.en.md

```diff
@@ -2,6 +2,8 @@
 
 As Figma currently provides no way to inspect plugin data as a viewer (we hope they do allow that at some point), you'd have to fall back to either letting your engineers use the plugin themselves and inspect layers, or use more traditional methods. One of those methods would be Redlining, where you'd create layers showing which tokens were applied to what layers, still containing token names so it would be clear what to apply when.
 
+<img src="/annotate.gif" alt="Image alt" />

```

Comment:
This graphic isn't loading for me, also Alt text should be explicit

Reply:
Done